ui/dashboards/src/components/Variables/VariableList.tsx

```diff
@@ -52,16 +53,19 @@ export function TemplateVariableList() {
           </Button>
         </Box>
       )}
-      <Box display={'flex'} justifyContent="space-between">
-        <Stack direction={'row'} spacing={2}>
-          {showVariables &&
-            variableDefinitions.map((v) => (
-              <Box key={v.spec.name} display={v.spec.display?.hidden ? 'none' : undefined}>
-                <TemplateVariable key={v.spec.name} name={v.spec.name} />
-              </Box>
-            ))}
-        </Stack>
-      </Box>
+
+      <AppBar color={'inherit'} position={trigger ? 'fixed' : 'static'} elevation={trigger ? 4 : 0}>
```

Comment:
Yeah +1 to Shan's comment, I really like this idea and it makes perfect sense in our app here, but with how we embed Perses internally, it will cause some problems since variables are further down in the component tree. Could an optional prop be added to turn it off just in case?

Reply:
Yeah I like more and more this idea. In terme of API exposed Iike the Idea you can customize the behaviour of the component with simple boolean properties.
I will do it !